contrib/gorilla/mux/mux.go

```diff
@@ -97,18 +97,31 @@ func (r *Router) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	var (
 		match    mux.RouteMatch
 		spanopts []ddtrace.StartSpanOption
-		route    = "unknown"
 	)
 	// get the resource associated to this request
 	if r.Match(req, &match) && match.Route != nil {
-		if r, err := match.Route.GetPathTemplate(); err == nil {
-			route = r
-		}
 		if h, err := match.Route.GetHostTemplate(); err == nil {
 			spanopts = append(spanopts, tracer.Tag("mux.host", h))
 		}
 	}
+	route := r.config.resourceNamer(r, req)
 	spanopts = append(spanopts, r.config.spanOpts...)
 	resource := req.Method + " " + route
 	httputil.TraceAndServe(r.Router, w, req, r.config.serviceName, resource, spanopts...)
 }
+
+// defaultResourceNamer attempts to quantize the resource for a HTTP request by
```

Comment:
```suggestion
// defaultResourceNamer attempts to quantize the resource for an HTTP request by
```
English isn't my primary language so please correct me if I'm wrong.

Reply:
Mine neither but I think you might be correct - I'll adjust :)